src/transformState.js

```diff
@@ -5,7 +5,49 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
look how the `switch case` is writing

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
> look how the `switch case` is writing
> 
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

I used curly brackets because I saw that this is how Yuriy writes `switch case`

Reply:
and thought that it would be better that way

Reply:
> look how the `switch case` is writing
> 
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

done